documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -3566,6 +3525,50 @@ The following configuration example adds the properties `database.connection.ada
 }
 ----
 
+[id="obtaining-oracle-jdbc-driver-and-xstreams-api-files"]
+=== Obtaining the Oracle JDBC driver and XStream API files
+
+The {prodname} Oracle connector requires the Oracle JDBC driver (`ojdbc8.jar`) to connect to Oracle databases.
+If the connector uses XStream to access the database, you must also have the XStream API (`xstreams.jar`).
+Licensing requirements prohibit {prodname} from including these files in the Oracle connector archive.
+However, the required files are available for free download as part of the Oracle Instant Client.
+The following steps describe how to download the Oracle Instant Client and extract the required files.
+
+.Procedure
+
+. From a browser, download the https://www.oracle.com/database/technologies/instant-client/downloads.html[Oracle Instant Client package] for your operating system.
+
+. Extract the archive, and then open the `instantclient___<version>__` directory.
++
+For example:
++
+[source]
+----
+instantclient_21_1/
+├── adrci
+├── BASIC_LITE_LICENSE
+├── BASIC_LITE_README
+├── genezi
+├── libclntshcore.so -> libclntshcore.so.21.1
+├── libclntshcore.so.12.1 -> libclntshcore.so.21.1
+
+...
+
+├── ojdbc8.jar
+├── ucp.jar
+├── uidrvci
+└── xstreams.jar
+
+----
+
+. Copy the `ojdbc8.jar` and `xstreams.jar` files, and add them to the `_<kafka_home>_/libs` directory, for example, `kafka/libs`.
```

Comment:
@Naros, is there a reason for copying that into kafka/libs rather the connector directory?

Reply:
@gunnarmorling @Naros Let me know if that should be changed. I brought that path forward from the current version of the docs.

Reply:
Right, I saw that. And we could go with that, but I'm curious about whether that's truly necessary. Waiting to hear back from @Naros.

Reply:
@gunnarmorling @roldanbob there is no specific reason; both are equally fine.  

The only thing that comes to mind is we may have chosen `kafka/libs` since its a static directory reference and would be universal across any type of Kafka Connect deployment whereas user-specific installations may actually specify the connector's plugin directory differently from our images versus others such as those from Confluent Hub.

FWIW, I did check the entire documentation and we do make the same call out in the Db2 documentation as well.

Reply:
So, bottom line, we can leave that as-is? 

Note that there will be further changes to the content change in this PR based on [DBZ-4392](https://issues.redhat.com/browse/DBZ-4392).

Reply:
Ok, logged https://issues.redhat.com/browse/DBZ-4883 as a follow-up and assigned it to you, @roldanbob. It describes some more changes which arguable could have been done as part of this PR already, but given it represents an improvement by itself, I went ahead and merged it, hoping we can get in those suggested follow-up changes soon, too. Thx!